weather@mockturtl/src/3_8/utils.ts

```diff
@@ -75,48 +75,95 @@ export function InjectValues(text: string, weather: WeatherData, config: Config,
-					.replace(/{{dew_point}}/g, Literal(dewPoint))
-					.replace(/{{humidity}}/g, Literal(humidity))
-					.replace(/{{pressure}}/g, Literal(pressure))
-					.replace(/{{pressure_unit}}/g, Literal(pressureUnit))
-					.replace(/{{extra_value}}/g, Literal(extraValue))
-					.replace(/{{extra_name}}/g, Literal(extraName))
-					.replace(/{{wind_speed}}/g, Literal(windSpeed))
-					.replace(/{{wind_dir}}/g, Literal(windDir))
-					.replace(/{{city}}/g, Literal(city))
-					.replace(/{{country}}/g, Literal(country))
-					.replace(/{{search_entry}}/g, Literal(searchEntry))
-					.replace(/{{last_updated}}/g, Literal(lastUpdatedTime))
-					.replace(/{{br}}/g, Literal("\n"));
-	}
-
-	return  text.replace(/{t}/g, temp)
-				.replace(/{u}/g, tempUnit)
-				.replace(/{c}/g, condition)
-				.replace(/{c_long}/g, conditionLong)
-				.replace(/{dew_point}/g, dewPoint)
-				.replace(/{humidity}/g, humidity)
-				.replace(/{pressure}/g, pressure)
-				.replace(/{pressure_unit}/g, pressureUnit)
-				.replace(/{extra_value}/g, extraValue)
-				.replace(/{extra_name}/g, extraName)
-				.replace(/{wind_speed}/g, windSpeed)
-				.replace(/{wind_dir}/g, windDir)
-				.replace(/{city}/g, city)
-				.replace(/{country}/g, country)
-				.replace(/{search_entry}/g, searchEntry)
-				.replace(/{last_updated}/g, lastUpdatedTime)
-				.replace(/{br}/g, "\n");
+	const tmr = weather.forecasts && weather.forecasts[1] ? weather.forecasts[1] : null;
+	const forecastHours = weather.hourlyForecasts && weather.hourlyForecasts[2] ? weather.hourlyForecasts : null;
+	const tempHourDiff = forecastHours ? ValueChange(TempToUserConfig(forecastHours[0].temp, config, false), TempToUserConfig(forecastHours[2].temp, config, false), false) : "";
+	const conditionTomorrow = tmr ? tmr.condition.main : "";
+	const tempMin = tmr ? TempToUserConfig(weather.forecasts[0].temp_min, config, false).toString() : "";
+	const tempMax = tmr ? TempToUserConfig(weather.forecasts[0].temp_max, config, false).toString() : "";
+	const tempMinTomorrow = tmr ? TempToUserConfig(tmr.temp_min, config, false).toString() : "";
+	const tempMaxTomorrow = tmr ? TempToUserConfig(tmr.temp_max, config, false).toString() : "";
+	const tempsTomorrow = tmr ? TempRangeToUserConfig(tmr.temp_min, tmr.temp_max, config) : "";
+	const tmrMinTempChange = tempMinTomorrow && temp ? SignedNumber(tempMinTomorrow - tempMin) : "";
+	const tmrMaxTempChange = tempMaxTomorrow && temp ? SignedNumber(tempMaxTomorrow - tempMax) : "";
+	const tempsTomorrowWithDifferences = tmr ? tempsTomorrow + " (" + tmrMinTempChange + " / " + tmrMaxTempChange + ")" : "";
+	const sunset = (GetHoursMinutes(weather.sunset, config._show24Hours));
+	const sunrise = (GetHoursMinutes(weather.sunrise, config._show24Hours));
+	const dayLength = ToHoursMinutes(weather.sunset - weather.sunrise);
+	const now = new Date();
+	const sunny = now > weather.sunrise && now < weather.sunset;
+	const daylightRemain = sunny ? ToHoursMinutes(weather.sunset - now) : "";
```

Comment:
This might be incorrect if the system timezone doesn't match the locations timezone (maybe it's all ok, I'm just not sure).

Please verify by testing a location outside your timezone.